site/troubleshooting.md

```diff
@@ -191,8 +191,7 @@ We sometimes publish snapshot releases, to provide previews of new
 features, assist in validation of bug fixes, etc. One can install the
 latest snapshot release with
 
-    sudo wget -O /usr/local/bin/weave \
-      https://raw.githubusercontent.com/weaveworks/weave/master/weave
+    sudo curl -L git.io/weave -o /usr/local/bin/weave
```

Comment:
Look where the original URL is pointing. It's *not* the latest release. So the short url is no use here.